### PR TITLE
refactor(card): tailwindify

### PR DESCRIPTION
**Pull request by julio8a**
Supports #1500

### CODE REVIEW

File: src/components/calcite-card/calcite-card.scss [hunk 1]
```diff
@@ -23,48 +26,41 @@
...
+    items-center
+    absolute
+    inset-0;
 }
 
 :host .header,
 :host .footer {
-  padding: $baseline/2;
-  display: flex;
+  @apply pt-3 pb-1 px-3 flex;
```
Comment (jrowlingson) [inline src/components/calcite-card/calcite-card.scss:45]: would p-3 work here?

File: src/components/calcite-card/calcite-card.scss [hunk 2]
```diff
@@ -73,53 +69,48 @@
...
-  @include font-size(-2);
 }
 @include slotted("footer-trailing", "*") {
-  align-self: center;
-  @include font-size(-2);
+  @apply self-center text--2-wrap;
 }
 
 :host .thumbnail-wrapper {
   font-size: 0;
```
Comment (jrowlingson) [inline src/components/calcite-card/calcite-card.scss:104]: :eyes:

Comment (julio8a) [reply src/components/calcite-card/calcite-card.scss:104]: I got caught!

**Comment by julio8a** [comment]
@bstifle, this is the card we worked on together, can you review screener?

**Review by jrowlingson — APPROVED** [review]

**Pull request merged**